libs/examples/src/examples.module.ts

```diff
@@ -138,7 +138,7 @@ import { DtExamplesTreeTableModule } from './tree-table/tree-table-examples.modu
     DtToastExamplesModule,
     DtToggleButtonGroupExamplesModule,
     DtExamplesTopBarNavigationModule,
-    DtExamplesTreeTableModule,
-  ],
+    DtExamplesTreeTableModule
```

Comment:
Does the linter not complain about missing trailing comma?
```suggestion
    DtExamplesTreeTableModule,
```